package.json

```diff
@@ -61,7 +61,7 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "express": "~4.16.4",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#task/lazyAttsNgsiv2-pr762",
```

Comment:
Revert before merging this PR, once telefonicaid/iotagent-node-lib#813 get merged

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/430/commits/5d23e76b41d22eb2a730cd8c14469cde4bceb2e3